src/main/scala/esmeta/ty/TyModel.scala

```diff
@@ -485,11 +491,36 @@ object TyModel {
         ),
       ),
       "ArrayBufferObject" -> TyInfo(parent = Some("Object")),
-      "BooleanObject" -> TyInfo(parent = Some("OrdinaryObject")),
-      "BigIntObject" -> TyInfo(parent = Some("OrdinaryObject")),
-      "NumberObject" -> TyInfo(parent = Some("OrdinaryObject")),
-      "SymbolObject" -> TyInfo(parent = Some("OrdinaryObject")),
-
+      "BooleanObject" -> TyInfo(
+        parent = Some("OrdinaryObject"),
+        fields = Map(
+          "BooleanData" -> (AbsentT || BoolT),
```

Comment:
nit-pick: I noticed that all fields added in this PR are marked with AbsentT. Is there a specific reason for this?